schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -1620,6 +1620,10 @@
             "copy": {
               "$ref": "#/definitions/resourceCopy"
             },
+            "scope": {
+              "type": "string",
+              "description": "Scope property to be used to provide scope for a resource or to be used to specify a scope for Management Group or Tenant level deployments."
```

Comment:
"to provide a scope for a resource" makes me think I can change the scope of any resource to any scope. Should we be more specific and say it works for two cases (today):
* setting the scope for extension resources
* deploying resources to the tenant scope in non-tenant scope deployments